src/notes.ml

```diff
@@ -258,55 +258,61 @@ value print_linked_list conf base pgl =
   tag "ul" begin
     List.iter
       (fun pg ->
-         stagn "li" begin
```

Comment:
Come on ! Why do you modify this code ? You simply change the indentation ! It makes really difficult to follow the modifications